project2.css

```diff
@@ -0,0 +1,396 @@
+header {
+  position: absolute;
+  width: 1340px;
+  height: 1246px;
+  left: 200px;
+  top: 71px;
+}
```

Comment:
As we need to display various items which may vary on the layout in the desktop version, I highly encourage you to use `display:grid` :checkered_flag:

Reply:
Also, not trying to be repetitive, but don't forget to [export the background images](https://help.figma.com/hc/en-us/articles/360040028114-Guide-to-exports-in-Figma) from [the Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) to use them in your project :star_struck: